src/casp/services/api/services/cellarium_general_service.py

```diff
@@ -94,7 +94,11 @@ def get_quota_for_user(self, user: models.User) -> schemas.UserQuota:
 
         return schemas.UserQuota(
             user_id=user.id,
-            quota=user.cell_quota,
-            remaining_quota=self.cellarium_general_dm.get_remaining_quota_for_user(user=user),
+            weekly_quota=user.cell_quota,
+            remaining_weekly_quota=self.cellarium_general_dm.get_remaining_quota_for_user(user=user),
             quota_reset_date=quota_reset_date,
+            lifetime_quota=user.lifetime_cell_quota,
+            remaining_lifetime_quota=(
```

Comment:
nit: Consider factoring out `None if user.lifetime_cell_quota is None else user.lifetime_cell_quota - user.total_cells_processed` into a separate variable. This could improve readability. For example:

```python3
remaining_lifetime_quota = (
    user.lifetime_cell_quota - user.total_cells_processed
    if user.lifetime_cell_quota is not None
    else None
)
```